testutils/inject/i2c.go

```diff
@@ -3,17 +3,17 @@ package inject
 import (
 	"context"
 
-	"go.viam.com/rdk/components/board"
+	"go.viam.com/rdk/components/board/genericlinux/buses"
```

Comment:
Right: we've moved where the interfaces are defined, but I didn't move where the injected implementation is, and this file doesn't use the implementation from a real board. If you're asking about testutils/inject/board.go instead, it doesn't mention I2C or SPI anywhere (because boards aren't in charge of those any more).